media/js/base/gtm/gtm-snippet.es6.js

```diff
@@ -0,0 +1,118 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+import {
+    consentRequired,
+    dntEnabled,
+    getConsentCookie,
+    gpcEnabled,
+    isFirefoxDownloadThanks
+} from '../consent/utils.es6';
+
+const GTM_CONTAINER_ID = document
+    .getElementsByTagName('html')[0]
+    .getAttribute('data-gtm-container-id');
+
+const GTMSnippet = {};
+
+/**
+ * Load the GTM snippet. Expects `GTM_CONTAINER_ID` to be
+ * defined in the HTML tag via a data attribute.
+ */
+GTMSnippet.loadSnippet = () => {
+    if (GTM_CONTAINER_ID) {
+        // prettier-ignore
+        (function(w,d,s,l,i,j,f,dl,k,q){
+            w[l]=w[l]||[];w[l].push({'gtm.start': new Date().getTime(),event:'gtm.js'});f=d.getElementsByTagName(s)[0];
+            k=i.length;q='//www.googletagmanager.com/gtm.js?id=@&l='+(l||'dataLayer');
+            while(k--){j=d.createElement(s);j.async=!0;j.src=q.replace('@',i[k]);f.parentNode.insertBefore(j,f);}
+        }(window,document,'script','dataLayer',[GTM_CONTAINER_ID]));
+    }
+};
+
+/**
+ * Determine if the current page is /firefox/download/thanks/.
```

Comment:
I think this needs to be tweaked to the Springfield-specific path, yeah?

```suggestion
 * Determine if the current page is /download/thanks/.
```

Reply:
Yeah the util itself looks right

https://github.com/mozmeao/springfield/blob/main/media/js/base/consent/utils.es6.js#L135